src/NuGet.Core/NuGet.Common/Errors/ILogFileContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NuGet.Common
+{
+    public interface ILogFileContext
```

Comment:
Add EndLineNumber and EndColumnNumber for completeness, the MSBuild takes those optional values also